samwise.js

```diff
@@ -1,4 +1,4 @@
-const doubledNumbers = numbers.map(num => num * 2);
```

Comment:
SCRIPT-The CSS rules here seem to be redundant. Can you please review and remove unnecessary ones?